netflix-roulette/src/pages/home/components/Catalog/components/MoviesList/MovieCard/MovieCard.styles.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+export const MovieCardContainer = styled.div`
+  flex-direction: column;
+  margin: 0 20px 40px 0;
+  color: #979797;
```

Comment:
Seems to be used multiple times, may be worth having as a part of your theme.

Reply:
Added, thanks